src/components/UI/search-box/search-box.module.scss

```diff
@@ -0,0 +1,11 @@
+.searchContainer {
+  display: flex;
+  align-items: center;
+  padding: 6px;
```

Comment:
NIT: Should we try for `rem` here instead?

Reply:
Sure we can use rem for padding if it is not going to affect during zoom of the page. Let me check with that.

Reply:
I don't think we need to use `rem` for padding and margin stuff
It's more useful with font-size